<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Setup</string>
     </attribute>
     <widget class="QComboBox" name="BeamlineSelect">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Beamline</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Plane</string>
      </property>
     </widget>
     <widget class="QComboBox" name="PlaneSelect">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>X</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Y</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="InformationLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>271</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string>Information</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ResetButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset tool and initialize</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="DryRunCheck">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>85</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Dry run</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>20</width>
        <height>491</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="StatusLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Status...</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="emit_norm_nm">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>500</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Assumed emittance (nm)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Displacement: min, max, step num
If steps=1, takes min value only</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="A_min">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>62</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="A_max">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>62</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
      <property name="value">
       <double>5.000000000000000</double>
      </property>
     </widget>
     <widget class="QSpinBox" name="A_steps">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>10</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Phase points in 2Ï€ range</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Phase_steps">
      <property name="geometry">
       <rect>
        <x>662</x>
        <y>70</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QPushButton" name="StartMeasurementButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start measurement</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>320</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>120</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Settle time (s)</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SettleTime">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>120</y>
        <width>62</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>5.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Measurement time per point (s)</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="MeasurementTime">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>160</y>
        <width>62</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>20.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="A_calc">
      <property name="geometry">
       <rect>
        <x>391</x>
        <y>340</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>340</y>
        <width>57</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>340</y>
        <width>57</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ïˆ (deg)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Phi_calc">
      <property name="geometry">
       <rect>
        <x>572</x>
        <y>340</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-180</number>
      </property>
      <property name="maximum">
       <number>179</number>
      </property>
     </widget>
     <widget class="QPushButton" name="CalcCorrAnglesButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>340</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Calc. Î”c</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RestoreCorrectorsButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>261</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Restore correctors to initial values</string>
      </property>
     </widget>
     <widget class="QLabel" name="CorrAngleResult">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>380</y>
        <width>441</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Result:</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>410</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="C0_calc">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>440</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="value">
       <double>0.000010000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="CalcAPhiButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>440</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Calc. A, Ïˆ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>440</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Î”c0 (mrad)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>440</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Î”c1 (mrad)</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="C1_calc">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>440</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="value">
       <double>0.000010000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="A_Phi_Result">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>480</y>
        <width>441</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Result:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Performance plots</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>751</width>
        <height>451</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Corrector plots</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
